cmd/hypervisor/commands/root.go

```diff
@@ -47,11 +49,15 @@ var rootCmd = &cobra.Command{
 		if _, err := buildinfo.Get().WriteTo(os.Stdout); err != nil {
 			log.Printf("Failed to output build info: %v", err)
 		}
-
```

Comment:
I think the new line here looks better because it split logical parts of the function: getting build info and preparing config.